src/plugins/term/queries.js

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 OpenStax Poland
+// Licensed under the MIT license. See LICENSE file in the project root for
+// full license text.
+
+/* type Term = {
+  reference: string,
+  leaf: Slate~Leaf,
+  focusText: Slate~Text,
+  offsetStart: number,
+  offsetEnd: number,
+} */
```

Comment:
You should define such types using JSDoc's virtual comments with [`@typedef`](http://usejsdoc.org/tags-typedef.html).